app/boards/shields/jiran/jiran-layouts.dtsi

```diff
@@ -1,6 +1,14 @@
+#include <layouts/kgoh/jian/6column.dtsi>
+#include <layouts/kgoh/jian/full.dtsi>
 #include <physical_layouts.dtsi>
 
-#include "position_map.dtsi"
+&layouts_kgoh_jian_position_map {
+    status = "disabled";
+};
```

Comment:
Why are we not just enhancing the existing map here, and duplicating things? Not sure I agree with this, unless there's some good reason to do so.